api/monetization.js

```
@@ -62,20 +62,24 @@ async function sendEmail(to, subject, token) {
 }
 
 // ─── Lazy-load Stripe (only when keys are available) ─────────────────────────
-function getStripe() {
+// Uses dynamic ESM import so the module doesn't crash when stripe is absent.
+async function getStripe() {
   const key = process.env.STRIPE_SECRET_KEY;
```

ESLint

[error] 'process' is not defined.

(no-undef)